test/test.dart

```diff
@@ -194,4 +196,34 @@ Future<void> setupBaas() async {
   final client = await (cluster == null ? BaasClient.docker(baasUrl) : BaasClient.atlas(baasUrl, cluster, apiKey!, privateApiKey!, projectId!));
 
   baasApps.addAll(await client.getOrCreateApps());
-}
\ No newline at end of file
+}
+
+@isTest
+Future<void> syncTest(
+  String name,
+  FutureOr<void> Function(ApplicationConfiguration configuration) testFunction, {
+  String appName = 'flexible',
+  dynamic skip,
+}) async {
+  final url = Uri.tryParse(Platform.environment['BAAS_URL'] ?? 'https://realm-dev.mongodb.com');
+  final apiKey = Platform.environment['BAAS_API_KEY'];
+  final projectId = Platform.environment['BAAS_PROJECT_ID'];
+
+  if (skip == null) {
+    skip = url == null || apiKey == null || projectId == null;
+  }
+  else if (skip is bool) {
```

Comment:
What is `skip` is a string?

Reply:
its passed to the test package

Reply:
its passed to the test package test method as is. Meaning skip is true (skip should happen) and no need to check for url